cookbooks/aws-parallelcluster-environment/kitchen.environment-config.yml

```diff
@@ -418,7 +418,7 @@ suites:
       - recipe[aws-parallelcluster-environment::mount_shared]
     verifier:
       controls:
-        - /tag:config_mount_shared/
+        - /mount_shared/
```

Comment:
[Nitpick] this always confuses me: can we try to use the full names of controls **unless** we specifically have a list of controls with the same initial pattern that we can use to specify multiple controls at once?